MonoGame.Framework/MathHelper.cs

```diff
@@ -266,29 +266,55 @@ public static float SmoothStep(float value1, float value2, float amount)
         /// <param name="radians">The angle in radians.</param>
         /// <returns>The angle in degrees.</returns>
         /// <remarks>
-        /// This method uses double precission internally,
+        /// This method uses double precision internally,
         /// though it returns single float
         /// Factor = 180 / pi
         /// </remarks>
         public static float ToDegrees(float radians)
         { 
             return (float)(radians * 57.295779513082320876798154814105);
         }
+
+        /// <summary>
+        /// Converts radians to degrees, returning a precise result.
+        /// </summary>
+        /// <param name="radians">The angle in radians.</param>
+        /// <returns>The angle in degrees.</returns>
+        /// <remarks>
+        /// Factor = 180 / pi
+        /// </remarks>
+        public static double ToDegreesD(double radians)
+        {
+            return (radians * 57.295779513082320876798154814105d);
+        }
         
         /// <summary>
         /// Converts degrees to radians.
         /// </summary>
         /// <param name="degrees">The angle in degrees.</param>
         /// <returns>The angle in radians.</returns>
         /// <remarks>
-        /// This method uses double precission internally,
+        /// This method uses double precision internally,
         /// though it returns single float
         /// Factor = pi / 180
         /// </remarks>
         public static float ToRadians(float degrees)
         { 
             return (float)(degrees * 0.017453292519943295769236907684886);
```

Comment:
Is there a good reason to not directly use a float here and get rid of the casting? Along with the `ToDegrees(float)` above?

Reply:
I'm not entirely sure, but with the more precise variants available, we should be safe to remove the casts.